test/index.test.js

```diff
@@ -39,20 +39,21 @@ describe("nc()", () => {
       .expect("quick math");
   });
 
-  it("supports async handlers", async () => {
+  it("supports async handlers and middleware", async () => {
```

Comment:
Can you add them as new test cases instead of editing existed ones?

Reply:
Got it 👍